dist/gcc-compatible/Hacl_Streaming_SHA2.c

```diff
@@ -39,7 +39,13 @@ Hacl_Streaming_SHA2_state_sha2_224 *Hacl_Streaming_SHA2_create_in_224()
         Hacl_Streaming_SHA2_state_sha2_224
       ));
   p[0U] = s;
-  Hacl_Hash_Core_SHA2_init_224(block_state);
+  KRML_MAYBE_FOR8(i,
```

Comment:
Would it be better not to inline `init`, `update_multi`, `update_last` and `finish`? We can define them as local functions for `Hacl.Streaming.SHA2`

Reply:
Done in 1ddc533